service/matching/tasklist/adaptive_scaler.go

```diff
@@ -345,19 +368,22 @@ func (a *adaptiveScalerImpl) describePartition(partitionID int) (*types.Describe
 func toAggregateMetrics(partitions map[int]*types.DescribeTaskListResponse) *aggregatePartitionMetrics {
 	total := 0.0
 	byIsolationGroup := make(map[string]float64)
+	pollersByIsolationGroup := make(map[string]int)
 	byPartition := make(map[int]*partitionMetrics, len(partitions))
 	for id, p := range partitions {
 		for ig, groupMetrics := range p.TaskListStatus.IsolationGroupMetrics {
 			byIsolationGroup[ig] += groupMetrics.NewTasksPerSecond
+			pollersByIsolationGroup[ig] += int(groupMetrics.PollerCount)
```

Comment:
I'm wondering if this number is accurate because a poller can poll 2 different partitions at different time, and we're not using any poller identity for deduplication, a poller may be counted twice.